### ISSUE TITLE
Module '"couchbase"' has no exported member

### ISSUE THREAD

**Issue by bwgjoseph** [bwgjoseph/mongoose-vs-ottoman#104]
Hi,

I have an issue compiling one of my [demo project](https://github.com/bwgjoseph/feathers-ottoman-demo). This demo project uses [feathers-ottoman](https://github.com/bwgjoseph/feathers-ottoman) package

You can clone the project, and run `npm run compile` and the following errors will occur

![image](https://user-images.githubusercontent.com/37484538/123990296-3939b000-d9fc-11eb-94ca-f78d627d67bc.png)

There's no issue running the project via `npm run dev` or running the test `npm run mocha` though.

Is there some issue with the way those exceptions are being exported?

Thanks

**Comment by gsi-alejandro** [comment]
hi @bwgjoseph 

try to set `skipLibCheck` to true in the tsconfig.json
![image](https://user-images.githubusercontent.com/15661442/124001518-b59fa500-d9a2-11eb-8f0f-929602a7532f.png)

This issue should be solved once the couchbase 3.2 version had been released, 3.2 will be shipped with typescript support.
Ottoman types will be improved after this release taking advantage of the SDK types.

**Comment by gsi-alejandro** [comment]
the couchbase 3.2 reimplemented the SDK in typescript: https://github.com/couchbase/couchnode/commit/59cbeddb9c5d43d09cff758c92ebd96627b806a2

**Comment by bwgjoseph** [comment]
Thanks, it does work with `skipLibCheck`

Is there any timeline for Couchbase 3.2 to be GA?

**Comment by gsi-alejandro** [comment]
@bwgjoseph 
The release date could be this July.

Am I correct @AV25242 ?

**Comment by bwgjoseph** [comment]
Thanks. Will keep this issue open until verified resolved.

**Comment by AV25242** [comment]
Assigning it back to you @bwgjoseph

**Comment by bwgjoseph** [comment]
This still doesn't work yet.

**Comment by gsi-alejandro** [comment]
hi @bwgjoseph 

This should work fine, but I need to check it.

we recently updated to couchbase SDK 3.2.x that provide correct types and should allow removing the skipLibCheck flag 

Later this will week I will notify you about this. (for now please keep using skipLibCheck or remove it if you want to check it by yourself)

**Comment by bwgjoseph** [comment]
Yup, I will test this a little later as well. Want to clear the existing issues first before testing this, probably few days later as well

**Comment by bwgjoseph** [comment]
Hi @gsi-alejandro,

I just ran a test by removing the `skipLibCheck` flag, and it still doesn't work but it seem much better now, only 2 errors

```ts
\feathers-ottoman-demo>npm run compile

> feathers-ottoman-demo@0.0.0 compile \feathers-ottoman-demo
> shx rm -rf lib/ && tsc

../feathers-ottoman/node_modules/couchbase/dist/logging.d.ts:1:19 - error TS7016: Could not find a declaration file for module 'debug'. 'Z:/Development/workspace/github/bwgjoseph/feathers-ottoman/node_modules/couchbase/node_modules/debug/src/index.js' implicitly has an 'any' type.       
  Try `npm i --save-dev @types/debug` if it exists or add a new declaration (.d.ts) file containing `declare module 'debug';`

1 import debug from 'debug';
                    ~~~~~~~

../feathers-ottoman/node_modules/ottoman/lib/types/model/model.d.ts:8:22 - error TS2420: Class 'Model<T, R>' incorrectly implements interface 'IModel<T, R>'.
  Type 'Model<T, R>' is missing the following properties from type 'IModel<T, R>': find, count, findById, findOne, and 13 more.

8 export declare class Model<T = any, R = any> extends Document implements IModel<T, R> {
                       ~~~~~


Found 2 errors.
```
